src/prisma/schema.prisma

```diff
@@ -2570,3 +2570,10 @@ model GitcoinProjectAttestation {
   schemaId       String
   attestationUID String
 }
+
+model ServiceWorkerSubscriptions {
+  id           String    @id @default(uuid()) @db.Uuid
+  createdAt    DateTime  @default(now())
+  deletedAt    DateTime?
+  subscription Json
```

Comment:
what goes in here? can we just use columns?

Reply:
Added user as optional since we can send a notification to a user or all the subscribers

Reply:
The json looks like this
```
const json = {
  endpoint: '.....',
  keys: {
    auth: '.....',
    p256dh: '.....'
  }
};

webpush.sendNotification(json, 'Your Push Payload Text');
```

I don't think 2 extra models will be helpful here since we don't use them at all. We just store and send it to web-push and it handles the rest.